CONTRIBUTING.md

```diff
@@ -43,21 +43,23 @@ we have a set of guidelines to ensure that all contributions are acceptable.
 
 To make the contribution process as seamless as possible, we ask for the following:
 
-* Fork the repository to work on your changes. Note that code contributions are accepted through pull requests to encourage discussion and allow for a smooth review experience.
-* When you’re ready to create a pull request, be sure to:
-  * Sign the [CLA](https://www.mongodb.com/legal/contributor-agreement).
-  * Have test cases for the new code. If you have questions about how to do this, please ask in your pull request or check the [Building and Testing](#building-and-testing) section.
-  * Run `make fmt`.
-  * Add documentation if you are adding new features or changing functionality.
-  * Confirm that `make check` succeeds. [GitHub Actions](https://github.com/mongodb/mongocli/actions).
+- Fork the repository to work on your changes. Note that code contributions are accepted through pull requests to encourage discussion and allow for a smooth review experience.
+- When you’re ready to create a pull request, be sure to:
+  - Sign the [CLA](https://www.mongodb.com/legal/contributor-agreement).
+  - Have test cases for the new code. If you have questions about how to do this, please ask in your pull request or check the [Building and Testing](#building-and-testing) section.
+  - Run `make fmt`.
+  - Add documentation if you are adding new features or changing functionality.
+  - Confirm that `make check` succeeds. [GitHub Actions](https://github.com/mongodb/mongocli/actions).
 
 ### Development Setup
 
 #### Prerequisite Tools
+
 - [Git](https://git-scm.com/)
-- [Go (at least Go 1.17)](https://golang.org/dl/)
```

Comment:
[flyby] Updating to match go.mod, the rest of the doc is auto formatted by my IDE